README.md

```diff
@@ -2,5 +2,62 @@
 
 Terraform module that will create an S3 bucket in AWS with relevant user account that will have access to bucket.
 
+The bucket created will use $team_name-$bucket_identifier as the bucket name to ensure the bucket created is globally unique and avoid name clashes with buckets.
+
 ## Usage
 
+```hcl
+module "example_team_s3" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=master"
+
+  team_name   = "example-repo"
+  bucket_name = "example-bucket"
+  acl         = "public-read"
+  versioning  = "true"
+
+
+}
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| acl | acl manages access to your bucket | string | `private` | no |
+| bucket_identifier | This is the bucket identifier, the bucket name will be this prefixed with your team name | string | - | yes |
+| team_name |  | string | - | yes |
+| versioning | version objects stored within your bucket. | string | `false` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| access_key_id | Access key id for s3 account |
+| bucket_arn | Arn for s3 bucket created |
+| bucket_name | bucket name |
+| iam_user_name | user name for s3 service account |
+| policy_arn | ARN for the new policy |
+| secret_access_key | Secret key for s3 account |
+| user_arn | Arn for iam user |
+
+## tags 
```

Comment:
Do we really want to specify all these tags as required variables to the bucket module? I think at least some of them can be automatically computed (eg. `owner`, `environment-name`) while some are perhaps not applicable to an S3 bucket (eg. `runbook`, `source-code`).

@kalbir thoughts?

Reply:
Spoke to @kalbir he suggested to have tags as variables that user inputs

removed source code/ runbook and component. Using var team_name for user